File: robotTutorial/src/my_robot_tutorials/scripts/my_first_node.py
#!/usr/bin/env python

import rospy

if __name__ == '__main__':
    rospy.init_node('first_python_node')
    rospy.loginfo('Node has started')
    rate = rospy.Rate(10)

    name = rospy.get_param("/robot_name")

    rospy.loginfo(name)

    while not rospy.is_shutdown():
        rospy.loginfo("Hello")
        rate.sleep()

File: robotTutorial/src/my_robot_tutorials/src/my_first_node.cpp
#include <ros/ros.h>
#include<string>
#include <my_robot_msgs/HardwareStatus.h>

using namespace std;

int main(int argc, char **argv)
{
    ros::init(argc, argv, "my_first_cpp_node");
    ros::NodeHandle nh;

    ROS_INFO("Node has been started");


    my_robot_msgs::HardwareStatus msg;

    string name;
    nh.getParam("/robot_name", name);

    cout << name << endl;

    ros::Rate rate(10);

    while (ros::ok)
    {
        ROS_INFO("Hello");
        rate.sleep();
    }
}
    
